openupgrade_scripts/scripts/hr_holidays/15.0.1.5/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,198 @@
+hr_holidays  / hr.leave.accrual.level   / first_day (integer)           : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / first_month (selection)       : NEW selection_keys: ['apr', 'feb', 'jan', 'jun', 'mar', 'may'], hasdefault
+hr_holidays  / hr.leave.accrual.level   / first_month_day (integer)     : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / frequency (selection)         : NEW required, selection_keys: ['bimonthly', 'biyearly', 'daily', 'monthly', 'weekly', 'yearly'], req_default: function, hasdefault
+hr_holidays  / hr.leave.accrual.level   / is_based_on_worked_time (boolean): NEW
+hr_holidays  / hr.leave.accrual.level   / maximum_leave (float)         : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / parent_id (many2one)          : NEW relation: hr.leave.accrual.level
+hr_holidays  / hr.leave.accrual.level   / second_day (integer)          : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / second_month (selection)      : NEW selection_keys: ['aug', 'dec', 'jul', 'nov', 'oct', 'sep'], hasdefault
+hr_holidays  / hr.leave.accrual.level   / second_month_day (integer)    : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / sequence (integer)            : NEW isfunction: function, stored
+hr_holidays  / hr.leave.accrual.level   / start_count (integer)         : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / start_type (selection)        : NEW required, selection_keys: ['day', 'month', 'year'], req_default: function, hasdefault
+hr_holidays  / hr.leave.accrual.level   / week_day (selection)          : NEW required, selection_keys: ['fri', 'mon', 'sat', 'sun', 'thu', 'tue', 'wed'], req_default: function, hasdefault
+hr_holidays  / hr.leave.accrual.level   / yearly_day (integer)          : NEW hasdefault
+hr_holidays  / hr.leave.accrual.level   / yearly_month (selection)      : NEW selection_keys: ['apr', 'aug', 'dec', 'feb', 'jan', 'jul', 'jun', 'mar', 'may', 'nov', 'oct', 'sep'], hasdefault
+# NOTHING TO DO: new model
+
+hr_holidays  / hr.leave.accrual.plan    / allocation_ids (one2many)     : NEW relation: hr.leave.allocation
+hr_holidays  / hr.leave.accrual.plan    / level_ids (one2many)          : NEW relation: hr.leave.accrual.level
+hr_holidays  / hr.leave.accrual.plan    / name (char)                   : NEW required
+hr_holidays  / hr.leave.accrual.plan    / time_off_type_id (many2one)   : NEW relation: hr.leave.type
+hr_holidays  / hr.leave.accrual.plan    / transition_mode (selection)   : NEW required, selection_keys: ['end_of_accrual', 'immediately'], req_default: function, hasdefault
+# NOTHING TO DO: new model
+
+hr_holidays  / hr.leave.allocation      / accrual_limit (integer)       : DEL
+hr_holidays  / hr.leave.allocation      / active (boolean)              : NEW hasdefault
+# NOTHING TO DO
+
+hr_holidays  / hr.leave.allocation      / accrual_plan_id (many2one)    : NEW relation: hr.leave.accrual.plan, isfunction: function, stored
+# NOTHING TO DO: handled by ORM
```

Comment:
You should pre-create the column for avoiding to trigger the compute method being False for all the records.